### PR TITLE
feat(preprocessing): add pre-classify command to manually classify files

### PR DESCRIPTION
**Pull request by 34j**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,7 +210,8 @@ Options:
 Commands:
   clean          Clean up files, only useful if you are using the default file structure
   infer          Inference
-  onnx           Export model to onnx
+  onnx
```
Comment (GarrettConway) [inline README.md:213]: did you mean to remove the onnx docstring?

Comment (34j) [reply README.md]: Wow, long time no see. ONNX support ended a long time ago, but I made a mistake in the order at that time.
https://github.com/voicepaw/so-vits-svc-fork/blob/df0dd614438a8fb1b4877b1f1bb5d4f0009db62c/src/so_vits_svc_fork/__main__.py#L766-L767

**Comment by 34j** [comment]
@GarrettConway Would you be a collaborator? (Becoming a collaborator does not create any obligation, but it does add one more emoji to your README while creating the possibility of increased mental pain)

**Comment by GarrettConway** [comment]
@34j Sure, though I don't know how much time I'll be able to commit to it in the long run. I was astounded by the number of changes you all have made while I was off on vacation and just recently caught up. Big kudos to you and the rest.

**Comment by 34j** [comment]
This repository is somewhat outdated and you should not spend too much time on it. (Although if I had more time, I would make all model parts interchangeable.) It is cursed because it has somewhat gained too much recognition.

**Pull request merged**